src/lib.rs

```diff
@@ -351,6 +351,15 @@ fn render_message(mut w: impl Write, config: &Config<'_>, msg: &Message, dbc: &D
         writeln!(&mut w, "}}")?;
         writeln!(w)?;
 
+        writeln!(&mut w, "/// Access message id")?;
+        writeln!(&mut w, "pub fn id(&self) -> u32 {{",)?;
+        {
+            let mut w = PadAdapter::wrap(&mut w);
+            writeln!(&mut w, "Self::MESSAGE_ID & 0x1FFF_FFFF")?;
```

Comment:
Just curious, why does it need the bitwise AND? If MESSAGE_ID is bigger than 29-bits then shouldn't it fail somewhere else?